sdk/src/logs/provider.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/metrics/provider.h"
+#include "opentelemetry/common/spin_lock_mutex.h"
+#include "opentelemetry/logs/logger_provider.h"
+#include "opentelemetry/logs/noop.h"
+#include "opentelemetry/logs/provider.h"
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/sdk/common/disabled.h"
+#include "opentelemetry/sdk/logs/provider.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+
+void Provider::SetLoggerProvider(
+    const nostd::shared_ptr<opentelemetry::logs::LoggerProvider> &lp) noexcept
+{
+  bool disabled = opentelemetry::sdk::common::GetSdkDisabled();
+
+  if (!disabled)
+  {
+    opentelemetry::logs::Provider::SetLoggerProvider(lp);
+  }
+}
+
+void Provider::SetEventLoggerProvider(
```

Comment:
Do we still need this? Seems evet has been removed from the spec?

Reply:
Thanks,

Removed all changes related to `EventLogger`, since this is going away.